package.json

```diff
@@ -29,7 +29,6 @@
 		"@nuxtjs/dotenv": "^1.4.0",
 		"@nuxtjs/pwa": "^3.0.0-beta.20",
 		"@nuxtjs/style-resources": "^1.0.0",
```

Comment:
Thanks for mentioning `@nuxtjs/style-resources` not being removed. I read [its README](https://github.com/nuxt-community/style-resources-module#readme). This package is to help with sharing variables, mixins, functions across all style files to get rid of the use of `@import`. Please let me know if my understanding is not complete.

However, our code base doesn't seem using this feature right now. It might make more sense to add this package when it's needed rather than leaving it without any usage as humans are forgetful. ;)

Reply:
True, I was just making sure I didn't remove overdo the removal.

Okay, will do this now.